src/bin/ion/commands/beta/inspect.rs

```diff
@@ -435,19 +408,26 @@ impl<'input> IonInspector<'input> {
     }
 
     fn write_field_if_present(&mut self) -> IonResult<()> {
-        if let Some(field_id) = self.reader.field_id() {
+        if let Some(field_token) = self.reader.raw_field_name_token() {
```

Comment:
The reader now has APIs that allow an application to see the original encoding of the field name, be it a symbol ID (`$12`) or text (`foo`).